webapp/src/main/webapp/vue-apps/matrix/components/message/MessageUploadFileInput.vue

```diff
@@ -89,17 +89,28 @@ export default {
       uploadProgress: 0,
       uploadedFiles: 0,
       ignoredFiles: 0,
+      pastTargetElement: null,
     };
   },
   props: {
     roomId: {
       type: String,
       default: null
+    },
+    pastTarget: {
```

Comment:
Could you update the name to paste instead of past ?

Reply:
Done